doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-commands-samples-ref.adoc

```diff
@@ -0,0 +1,876 @@
+      }
+    }
+  ]
+}
+----
+--
+GetServerStateCommand::
++
+--
+Returns information about the current state and configurations of the {KIE_SERVER}.
+
+.Command attributes
+[cols="30%,50%,20%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Requirement
+
+|
+|
+|Required
+
+|
+|
+|Optional
+|===
+
+.Example Java command
+[source,java]
+----
+
+----
+
+.Example REST request body (JSON)
+[source,json]
+----
+{
+  "commands" : [ {
+    "get-server-state" : { }
+  } ]
+}
+----
+
+.Example server response (JSON)
+[source,json,subs="attributes+"]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "Successfully loaded server state for server id default-kieserver",
+      "result": {
+        "kie-server-state-info": {
+          "controller": [
+            "http://localhost:8080/{URL_COMPONENT_CENTRAL}/rest/controller"
+          ],
+          "config": {
+            "config-items": [
+              {
+                "itemName": "org.kie.server.location",
+                "itemValue": "http://localhost:8080/kie-server/services/rest/server",
+                "itemType": "java.lang.String"
+              },
+              {
+                "itemName": "org.kie.server.controller.user",
+                "itemValue": "controllerUser",
+                "itemType": "java.lang.String"
+              },
+              {
+                "itemName": "org.kie.server.controller",
+                "itemValue": "http://localhost:8080/{URL_COMPONENT_CENTRAL}/rest/controller",
+                "itemType": "java.lang.String"
+              }
+            ]
+          },
+          "containers": [
+            {
+              "container-id": "employee-rostering",
+              "release-id": {
+                "group-id": "employeerostering",
+                "artifact-id": "employeerostering",
+                "version": "1.0.0-SNAPSHOT"
+              },
+              "resolved-release-id": null,
+              "status": "STARTED",
+              "scanner": {
+                "status": "STOPPED",
+                "poll-interval": null
+              },
+              "config-items": [
+                {
+                  "itemName": "KBase",
+                  "itemValue": "",
+                  "itemType": "BPM"
+                },
+                {
+                  "itemName": "KSession",
+                  "itemValue": "",
+                  "itemType": "BPM"
+                },
+                {
+                  "itemName": "MergeMode",
+                  "itemValue": "MERGE_COLLECTIONS",
+                  "itemType": "BPM"
+                },
+                {
+                  "itemName": "RuntimeStrategy",
+                  "itemValue": "SINGLETON",
+                  "itemType": "BPM"
+                }
+              ],
+              "messages": [],
+              "container-alias": "employeerostering"
+            }
+          ]
+        }
+      }
+    }
+  ]
+}
+----
+--
+CreateContainerCommand::
```

Comment:
container attribute of type KieContainerResource is mandatory

Reply:
Done.